app/models/samples/energyconstructionsetabridged.py

```diff
@@ -0,0 +1,60 @@
+construction_set = {
+    'type': 'ConstructionSetAbridged',
+    'name': 'Construction Set',
+    'wall_set': {
+        'type': 'WallSetAbridged',
```

Comment:
Can a ConstructionSetAbridged contain a non-abridged WallSet?

Reply:
No, it cannot.  The Abridged schema and the non-Abridged schema are two separate worlds of objects.  The only place where their paths cross are for objects that do not possess other nested objects (ie. materials)